eo-maven-plugin/src/main/java/org/eolang/maven/SafeMojo.java

```diff
@@ -220,6 +241,13 @@ abstract class SafeMojo extends AbstractMojo {
     @SuppressWarnings("PMD.ImmutableField")
     private boolean skip;
 
+    /**
```

Comment:
@maxonfjvipon Do we need a JavaDoc for the method which we `@Override`?

Reply:
@volodya-lombrozo the reason of that is necessity of using `@checkstyle CyclomaticComplexityCheck (70 lines)`.